.pre-commit-config.yml

```diff
@@ -0,0 +1,4 @@
+-   repo: github.com/gruntwork-io/pre-commit
+    sha: HEAD
```

Comment:
Change to `rev: v0.0.2`

Reply:
oops, missed this in the last update. Adding that now